lib/OpenQA/Scheduler.pm

```diff
@@ -689,7 +693,6 @@ sub _job_skip_children{
         {
             state => OpenQA::Schema::Result::Jobs::DONE,
             result => OpenQA::Schema::Result::Jobs::INCOMPLETE,
-            t_started => now(),
             t_finished => now(),
```

Comment:
AFAIR scheduled jobs are meant to be set to cancelled rather than done+incomplete. the latter is uses for jobs that were running already. And in case of cancelled neither t_started nor t_finished make sense to be set.